test/system/src/test/java/io/pravega/test/system/AbstractReadWriteTest.java

```diff
@@ -552,6 +573,89 @@ private EventWriterConfig buildWriterConfig() {
         return validEvents;
     }
 
+    void createScope(String scopeName, StreamManager streamManager) {
+        Boolean createScopeStatus = streamManager.createScope(scopeName);
+        if (createScopeStatus) {
+            log.info("Successfully created Scope {}", scopeName);
+        }
+    }
+
+    void createKVT(String scopeName, String kvtName, KeyValueTableConfiguration config, URI controllerURIDirect) {
+        log.info("Creating KVT name:{} in scope {}", kvtName, scopeName);
+        Boolean createKvtStatus = KeyValueTableManager.create(controllerURIDirect).createKeyValueTable(scopeName, kvtName, config);
+        if (createKvtStatus) {
```

Comment:
What happens if this is false? Shall we throw an exception and fail the test?

Reply:
resolved, createKeyValueTable will give the exception and will fail the test , verify true to print success message